### DISCUSSION TITLE
MySQLBACKUP EXPORTTOFILE

### DISCUSSION THREAD

**Discussion by loganhedges48** [MySqlBackupNET/MySqlBackup.Net#104] [General]
Hello I was wondering if there was a way to use ExportToFile but instead of it dumping all the tables, it selected a one specific table at a time and exported it.

Thanks.

**Comment by adriancs2** [comment]
You can declare a list of tables that you wish to export, perform a loop and use the option `ExportInfo.TablesToBeExportedDic` to do the backup.
https://github.com/MySqlBackupNET/MySqlBackup.Net/wiki/Conditional-Rows-Export-for-Each-Table

**Comment by adriancs2** [comment]
Here's an example of exporting each table into separate individual backup file. Read more at the following wiki page:

https://github.com/MySqlBackupNET/MySqlBackup.Net/wiki/Export-Each-Table-As-Separate-SQL-Backup-File
